src/k8s-extension/azext_k8s_extension/commands.py

```diff
@@ -20,6 +20,6 @@ def load_command_table(self, _):
                             is_preview=True) \
             as g:
         g.custom_command('create', 'create_k8s_extension')
-        g.custom_command('delete', 'delete_k8s_extension', confirmation=True)
+        g.custom_command('delete', 'delete_k8s_extension')
```

Comment:
May I ask why do you delete the confirmation?

Reply:
We want to allow custom logic for our partners to have confirmations or not and to print different kinds of messages based on the extension_type. We have moved the confirmation logic into the `user_confirmation_factory`

Reply:
Got it, thanks~